projects/packages/forms/src/blocks/contact-form/components/jetpack-integrations-modal/index.tsx

```diff
@@ -78,62 +29,13 @@ const IntegrationsModal = ( {
 			className="jetpack-forms-integrations-modal"
 		>
 			<VStack spacing="4">
```

Comment:
To me, `VStack` would make sense inside `IntegrationsList`, where we produce the list it controls visually as well.